netbox/templates/dcim/cable_trace.html

```diff
@@ -88,6 +88,10 @@ <h4 class="text-muted">No Cable</h4>
                     </table>
                 </div>
             </div>
+        {% elif position_stack %}
+            <div class="col-md-11 col-md-offset-1">
+                <h3 class="text-warning text-center">Multiple possible paths end at this point. No connection established!</h3>
+            </div>
```

Comment:
When tracing from FrontPort to RearPort there may be an endpoint, but it can/will be the endpoint from multiple traces. It is therefore not considered a connected endpoint, so explain this to the user.

Reply:
As with `split_ends`, the user should be presented options at this point. How do we convey the possible next paths?

Reply:
If there is something left on `position_stack` that indicates that there are no possible next paths. There is no choice to be made. We can only notify the user that the path ends sooner than expected.